src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,36 @@
+import { Navigate, NavLink, useLocation } from 'react-router-dom';
+import cn from 'classnames';
+import { Path } from '../../types';
+
+const getNavStyles = ({ isActive }: { isActive: boolean }) =>
+  cn('navbar-item', { 'has-background-grey-lighter': isActive });
+
+export const Navigation = () => {
+  const location = useLocation();
+  const { pathname } = location;
+
+  if (pathname === `/${Path.home}`) {
+    return <Navigate to={Path.main} replace={true} />;
+  }
```

Comment:
You don't need to navigate like this, you can add a route that will do it, something like this
```tsx
<Route path="home" element={<Navigate to="/" replace />} />
```